applications/CoSimulationApplication/python_scripts/factories/helpers.py

```diff
@@ -34,8 +34,15 @@ def CreateConvergenceAccelerators(convergence_accelerator_settings_list: KM.Para
                                   parent_echo_level: int):
     convergence_accelerators = []
     for conv_acc_settings in convergence_accelerator_settings_list.values():
-        solver = solvers[conv_acc_settings["solver"].GetString()]
-        interface_data_dict = solver.data_dict
+        if conv_acc_settings.Has("solver"):
+            solver = solvers[conv_acc_settings["solver"].GetString()]
+            interface_data_dict = solver.data_dict
+        else:
+            interface_data_dict = {}
+            for sequenceData in conv_acc_settings["solver_sequence"].values():
+                solver = solvers[sequenceData["solver"].GetString()]
+                data_name = sequenceData["data_name"].GetString()
+                interface_data_dict[data_name] = solver.data_dict[data_name]
```

Comment:
this becomes problematic if more things are added, thus my suggestion above to make the check more explicit